src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderImageRepository.java

```diff
@@ -17,7 +16,7 @@
     List<OffenderImage> getImagesByOffenderNumber(@Param("offenderNumber") final String offenderNumber);
 
     @Query(value = "select oi from OffenderImage oi where oi.id = (SELECT max(i.id) FROM OffenderImage i join i.offenderBooking ob join ob.offender o where o.nomsId = :offenderNumber " +
-        "and i.viewType = 'FACE' and i.orientationType = 'FRONT' and i.imageType = 'OFF_BKG' and i.active = true)")
+        "and ob.bookingSequence = 1 and i.viewType = 'FACE' and i.orientationType = 'FRONT' and i.imageType = 'OFF_BKG' and i.active = true)")
     Optional<OffenderImage> findLatestByOffenderNumber(@Param("offenderNumber") final String offenderNumber);
```

Comment:
This (I think) fixes the issue where old images (from old bookings) were being returned when accessing images by `offenderNumber`. Accessing by `bookingId` would work ok.

Reply:
Suspect in prod its not an issue